Css/WebScrap.Css/CssValidator.cs

```diff
@@ -0,0 +1,15 @@
+namespace WebScrap.Css;
+
+public static class CssValidator
+{
+    public static void ThrowIfUnsupported(
+        ReadOnlySpan<char> css, 
+        ReadOnlySpan<char> bannedChars)
```

Comment:
I think this file is a good place for unsupported registry.
Please remove `bannedChars` from public view.
Do dedicated methods with clear registry of chars instead.

```
ThrowIfAttributesUnsupported(css) => ThrowIfUnsupported(css, "[]")
```